CHANGELOG/v1.8.0-rc.0.md

```diff
@@ -0,0 +1,457 @@
+🚨 This is a RELEASE CANDIDATE. Use it only for testing purposes. If you find any bugs, file an [issue](https://github.com/kubernetes-sigs/cluster-api/issues/new).
+
+## Highlights
```

Comment:
Please add:
* #10950
* #10952

Reply:
Added

Reply:
Sorry the idea wasn't to add these two PRs to highlights (I just put the comment somewhere). Please move them to the correct sections (based on the PR icon)

Reply:
@vishalanarase ^^